py3status/modules/group.py

```diff
@@ -50,6 +50,8 @@
 
 Format placeholders:
     {button} The button to open/close or change the displayed group
+    {index} The module index
```

Comment:
this is incorrect.

Py3status modules effectively have 3 names here with examples (i3status modules do not quite follow this but could)

```
# partial config
github {}
github some_project {}
group {
    github {}
}
```
these modules then look like

__module_full_name__
github
github some_project
github _anon_module_7

__module_name__
github
github
github

__module_nice_name__
github
github some_project
github


`index` implies that it is to do with the position in the group.

But the main idea behind the PR is sound

Reply:
I see it does not always print number. Switched to `{anon}`. Fixed.

Reply:
would not something like `{module}`, `{module_full}`, `{module_type}` be simpler because that actually makes sense `{anon}` does not really, internally it has no meaning either, it is just part of the modules name

Reply:
Review, please.